### PR TITLE
#163085904 write tests for user routes

### PR DESCRIPTION
**Pull request by Easybuoy**
#### What does this PR do?
Write tests for user routes.

#### Description of Task to be completed?
- Create a file <code>user.test.js</code>
- Use <code>chai, chai-http</code> to write tests for <code>user</code> routes.
- Use <code>expect</code> to assert tests.

#### How should this be manually tested?
- run <code>npm test</code>
- all <code>tests</code> should fail.

#### Any background context you want to provide?
none

#### What are the relevant pivotal tracker stories?
#163085904

#### Screenshots (if appropriate)
none

#### Questions:
none

### CODE REVIEW

**Comment by Easybuoy** [comment]
> I'd like you to create individual describe blocks inside the main describe block.
> 
> These describe blocks should handle each route you are referring to.
> 
> As an example, the register route should be something like:
> 
> `describe('Register Route: /api/v1/users/register', () => ...)`

Okay, would work on it now.

**Comment by Easybuoy** [comment]
> > I'd like you to create individual describe blocks inside the main describe block.
> > These describe blocks should handle each route you are referring to.
> > As an example, the register route should be something like:
> > `describe('Register Route: /api/v1/users/register', () => ...)`
> 
> Okay, would work on it now.

I've pushed the requested changes.

File: test/user.test.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+
+import app from '../app';
+
+chai.use(chaiHttp);
+
+const { expect } = chai;
+
+describe('Register Route: /api/v1/users/register', () => {
```
Comment (oreoluwade) [inline test/user.test.js:10]: You're almost there. Wrap all these `describe` blocks in a single `describe` block. It means you'd end up having one large describe and children describe blocks.

**Comment by oreoluwade** [comment]
You should test for negatives as well, not only positives. So you should include test cases for when login will fail, register will fail and so on.

File: test/user.test.js (hunk 1 above)
Comment (Easybuoy) [reply test/user.test.js:10]: okay.

**Pull request closed without merging**